From 073873cb152cf254d229d548c3bdcc57460a4ff0 Mon Sep 17 00:00:00 2001
From: Mikulas Patocka <mpatocka@redhat.com>
Date: Thu, 1 Mar 2018 21:17:19 -0500
Subject: [PATCH 2/2] brd: remove unused brd_mutex

commit 15f7b41f70ddcca3b555bd0fdc7c8da7466b517e upstream.

Remove unused mutex brd_mutex. It is unused since the commit ff26956875c2
("brd: remove support for BLKFLSBUF").

Signed-off-by: Mikulas Patocka <mpatocka@redhat.com>
Signed-off-by: Jens Axboe <axboe@kernel.dk>
Signed-off-by: Liwei Song <liwei.song@windriver.com>
Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 drivers/block/brd.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/block/brd.c b/drivers/block/brd.c
index ef049e3c9709..d01d84314b16 100644
--- a/drivers/block/brd.c
+++ b/drivers/block/brd.c
@@ -58,7 +58,6 @@ struct brd_device {
 /*
  * Look up and return a brd's page for a given sector.
  */
-static DEFINE_MUTEX(brd_mutex);
 static struct page *brd_lookup_page(struct brd_device *brd, sector_t sector)
 {
 	pgoff_t idx;
-- 
2.5.0

